inference-engine/src/mkldnn_plugin/nodes/mkldnn_input_node.cpp

```diff
@@ -68,6 +403,18 @@ MKLDNNInputNode::MKLDNNInputNode(const InferenceEngine::SizeVector &dims, const
     }
 }
 
+void MKLDNNInputNode::withMeanImage() {
+    isMeanImage = true;
+}
+
+const InferenceEngine::Blob::CPtr MKLDNNInputNode::getConstBlob() const {
+    return constBlob;
+}
+
+MKLDNNMemoryPtr MKLDNNInputNode::getMemoryPtr() const {
```

Comment:
Having both `getMemoryPtr` and `getConstBlob` looks like API duplication. Moreover using getConstBlob() might lead to denormals spread throughout the plugin. Can we consider removing `getConstBlob`?

Reply:
ok, i'll remove it